src/Database/PostgreSQL/Simple/Interval.hs

```diff
@@ -0,0 +1,241 @@
+{-# LANGUAGE LexicalNegation #-}
+{-# LANGUAGE NumDecimals #-}
+
+module Database.PostgreSQL.Simple.Interval where
```

Comment:
This module exposes lots of utility functions that probably shouldn't be part of the public API. If desired, I could move this into an internal module and only re-export the interesting stuff from here.

Reply:
Having an explicit export list would be a good start even before I look any further.